index.bs

```diff
@@ -777,9 +777,9 @@ The following shared conditions and requirements from [[!MIAF]] should apply:
 The following additional constraints apply to all [=AV1 Image Items=] and all [=AV1 Image Sequences=]:
     - <assert>The AV1 profile shall be the Main Profile and the level shall be 5.1 or lower.</assert>
 
-        NOTE:  AV1 tiers are not constrained because timing is optional in image sequences and are not relevant in image items or collections.
+        NOTE:  AV1 tiers are not constrained because timing is optional in image sequences and is not relevant in image items or collections.
 
-        NOTE:  Level 5.1 is chosen for the Baseline profile to ensure that no single coded image exceeds 4k resolution, as some decoder may not be able to handle larger images. More precisely, following [[!AV1]] level definitions, coded image items compliant to the [=AVIF Baseline profile=] may not have a number of pixels greater than 8912896, a width greater than 8192 or a height greater than 4352. It is still possible to use the Baseline profile to create larger images using a [=grid derived image item=].
+        NOTE:  Level 5.1 is chosen for the Baseline profile to ensure that no single coded image exceeds 4k resolution, as some decoders may not be able to handle larger images. More precisely, following [[!AV1]] level definitions, coded image items compliant to the [=AVIF Baseline profile=] may not have a number of pixels greater than 8912896, a width greater than 8192 or a height greater than 4352. It is still possible to use the Baseline profile to create larger images using a [=grid derived image item=].
```

Comment:
Changed "some decoder" to "some decoders" (plural) because we are not referring to a certain decoder here.

Reply:
The 'decoder' to 'decoders' change looks good to me.

I think '4k' should be '8k' no? Since in the next section it describes a 8k resolution. Level 3 is ~4K (4352x2448).

Reply:
I also wondered about that. It turns out that for each level, the maximum number of pixels is more restrictive than the maximum width or height. So one possible explanation I came up with for "no single coded image exceeds 4k resolution" is that the maximum number of pixels 8912896 (= 4352 * 2048)  is close to 4k resolution.

But even with this explanation, 8912896  > 4126 x 2160, which exceeds [4k resolution](https://en.wikipedia.org/wiki/4K_resolution). So the statement that at level 5.1 "no single coded image exceeds 4k resolution" is incorrect.

Reply:
Thanks for the suggestion! We can write a separate pull request to fix that.

By the way, I checked the levels around level 5.1 more closely. I found that no level supports exactly 4k resolution, but levels 5.0 - 5.3 are the closest to 4k resolution. Among these levels, it's not clear why level 5.1 was chosen. It may have to do with the frame rate limit.

Reply:
I filed https://github.com/AOMediaCodec/av1-avif/issues/290 for this issue.